images/chromium-headful/client/src/neko/base.ts

```diff
@@ -241,18 +263,27 @@ export abstract class BaseClient extends EventEmitter<BaseEvents> {
           break
         case 'connected':
           this.onConnected()
+          // Start WebRTC stats collection
+          if (this._peer) {
+            this._webrtcStatsCollector.start(this._peer)
```

Comment:
@raiden-staging just double checking my understanding--the client would always be collecting these stats and sending them to the server, and then when you hit the `/dev/benchmark` endpoint it observes the incoming metrics from all connected webrtc clients and computes stats?